draft-js-inline-toolbar-plugin/src/components/Toolbar/index.js

```diff
@@ -4,6 +4,7 @@ import { getVisibleSelectionRect } from 'draft-js';
 
 // TODO make toolbarHeight to be determined or a parameter
 const toolbarHeight = 44;
+const padding = 5;
```

Comment:
what is this?

Reply:
@juliankrispel I've attached screenshots of the issue. Hope it explains.
Let me know if you see an issue with the approach.

Reply:
attached more screenshots to the pr